src/library/Uno.Themes.WinUI.Markup/Theme/ToggleButton.cs

```diff
@@ -0,0 +1,379 @@
+﻿using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class ToggleButton
+		{
+			public static class Resources
+			{
+				public static class TextToggleButton
```

Comment:
We should remove the "ToggleButton" part of this static class so it's just `public static class Text` and `public static class Icon`

Same way we split the Resources class for Button like this:

![image](https://github.com/unoplatform/Uno.Themes/assets/4793020/43e8b4cf-b086-408e-ab2a-52cb8eafdeef)

Reply:
fixed